.baseline/checkstyle/checkstyle.xml

```diff
@@ -296,11 +296,6 @@
             <property name="format" value="\? extends Object\W"/>
             <property name="message" value="Use ? rather than ? extends Object."/>
         </module>
-        <module name="RegexpSinglelineJava">
```

Comment:
I don't think we should remove this. It catches many bugs that vanilla Guava users produce.

Reply:
oh, you replaced it with error-prone. got it.